editor/docks/editor_dock.cpp

```diff
@@ -105,7 +105,6 @@ void EditorDock::_bind_methods() {
 }
 
 EditorDock::EditorDock() {
-	set_clip_contents(true);
```

Comment:
Originally suggested in https://github.com/godotengine/godot/pull/106503#issuecomment-2891907897, but wasn't really discussed.

Note that various docks were disabling it, so these lines need to be removed if we go with this change:
https://github.com/godotengine/godot/blob/dec5a373d97cbadce5e1fabe6095ae9957ff3aa6/editor/debugger/editor_debugger_node.cpp#L73
https://github.com/godotengine/godot/blob/dec5a373d97cbadce5e1fabe6095ae9957ff3aa6/editor/script/find_in_files.cpp#L1304
https://github.com/godotengine/godot/blob/dec5a373d97cbadce5e1fabe6095ae9957ff3aa6/modules/mono/editor/GodotTools/GodotTools/Build/MSBuildPanel.cs#L211